sdk/eventhub/Azure.Messaging.EventHubs/tests/Compatibility/TrackOneSharedAccessTokenTests.cs

```diff
@@ -35,7 +35,7 @@ public void ConstructorValidatesTheSignature()
         [Test]
         public void ConstructorValidatesTheSignatureValue()
         {
-            var signature = new SharedAccessSignature("audience", "keyName", "key", null, DateTime.UtcNow);
+            var signature = new SharedAccessSignature("audience", "keyName", "key", null, DateTimeOffset.Now);
```

Comment:
These should be `DateTimeOffset.UtcNow.`    Not a huge deal in the tests, but `UtcNow` = the UTC offset (nothing) where `Now` uses the local timezone offset.  

[reference](https://docs.microsoft.com/en-us/dotnet/api/system.datetimeoffset.now?view=netframework-4.8)

Reply:
Done.